modules/client/src/test/java/org/apache/ignite/client/ClientKeyValueBinaryViewTest.java

```diff
@@ -89,6 +89,19 @@ public void testPutGet() {
         assertTupleEquals(val, resVal);
     }
 
+    @Test
+    public void testPutNull() {
+        Table table = defaultTable();
+        KeyValueView<Tuple, Tuple> kvView = table.keyValueView();
+
+        Tuple key = defaultTupleKey();
+
+        kvView.put(null, key, null);
```

Comment:
null-tuple is not allowed.

Reply:
It is allowed, please see https://github.com/apache/ignite-3/blob/825a20586414c5d5bbf38686ccd3de54c9189b48/modules/table/src/test/java/org/apache/ignite/internal/table/KeyValueBinaryViewOperationsTest.java#L84